huxley/accounts/models.py

```diff
@@ -15,19 +15,24 @@ class User(AbstractUser):
 
     TYPE_ADVISOR = 1
     TYPE_CHAIR = 2
+    TYPE_DELEGATE = 3
     USER_TYPE_CHOICES = ((TYPE_ADVISOR, 'Advisor'),
-                         (TYPE_CHAIR, 'Chair'))
+                         (TYPE_CHAIR, 'Chair'),
+                         (TYPE_DELEGATE, 'Delegate'))
```

Comment:
Yes! I think we need to discuss the model framework I am thinking about for the Delegate users.